src/Routes/Sign-Up/Sign-up.jsx

```diff
@@ -27,13 +32,33 @@ export function SignUp(){
           password_confirmation: confirmPassword
         }
       });;
-      //TO DO: mostrar mensaje de registro exitoso y luego esperar x segundos para dirigir al sign in
-      navigate("/SignIn");
+
+      toast.success("User registered successfully!", {
+        autoClose: 1500, 
+      });
+      
+      const responseSignIn = await axiosInstance.post("/users/sign_in", {
+        user: {
+          email,
+          password,
+        }
+      });
+      const accessToken = responseSignIn.headers['access-token'];
+      const uid = responseSignIn.headers['uid'];
+      const client = responseSignIn.headers['client'];
+      
+      localStorage.setItem('accessToken', accessToken);
+      localStorage.setItem('uid', uid);
+      localStorage.setItem('client', client);
+
+      handleLogin();
+      // We should navigate to the HomePage or to the sign in?
```

Comment:
This is your call, I would redirect to HomePage as you are already setting user credentials in local storage.

Reply:
I agree, and Lucia also recommended it. However, the project brief mentions that users should be redirected to the login page after registering. I think this might be unnecessary.

Should we strictly adhere to the brief's instructions, or can we redirect users to the homepage instead and avoid having them log in since we already have their credentials?